docs/elements/volusion/tips.md

```diff
@@ -11,13 +11,13 @@ order: 50
 sitemap: false
 ---
 
-## Volusion Tips
+# Tips
 
-How does Volusion Pagination work?
+The Tips section includes more information about the element including FAQs, endpoint limitations, and general tips.
 
-Volusion uses a sync style API for the GET /products and GET /orders resources.
+* Volusion uses a sync style API for the `GET /products` and `GET /orders` resources.
 This means that pagination will automatically occur during each request, and
-must be reset through the Volusion website. For example, a call to GET /products will
+must be reset through the Volusion website. For example, a call to `GET /products` will
 fetch the first page of 100 results, and then the next call will fetch the second page of 100
 results. This will continue to occur until the entire store of results has been
 returned. There is no need to specify a `page` or `pageSize` parameter.
```

Comment:
I know you didn't write the part below, but could you change it to:

To reset pagination:

1. From the Volusion Admin Dashboard, select **Inventory Import/Export > Volusion API > RUN Generic/Orders**.
2. Select a date from the list.
3. Click **Reset Export**.